### PR TITLE
Add jupyter notebook for supercritical power plant and supercritical steam cycle process

### PR DESCRIPTION
**Pull request by luohezhiming**
# Title

Migrate `supercritical_power_plant.ipynb` and `supercritical_steam_cycle.ipynb` from https://github.com/IDAES/examples-pse/tree/main/src/Examples/Flowsheets/power_generation/supercritical

----
Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

    I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
    I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

----
:books: Documentation preview :books:: https://idaes-examples--39.org.readthedocs.build/en/39/

### CODE REVIEW

File: idaes_examples/notebooks/docs/surrogates/alamo_run.alm
Comment (andrewlee94) [inline idaes_examples/notebooks/docs/surrogates/alamo_run.alm:23]: You seem to have included some files you shouldn't have. This is probably due to running tests (which indicates we should fix that too), but for this PR you should not include these in the commit.

Comment (luohezhiming) [reply idaes_examples/notebooks/docs/surrogates/alamo_run.alm:23]: I am not so sure if the test or build documentation lead to this but I believe I should not include this. But I find I no matter how I modify the this `alamo_run.alm` file, it is not tracked and I cannot commit the modification related to this file. When I check the status, it does not show the modificaitons of this file.

**Comment by lbianchi-lbl** [comment]
@dangunter are the files in `nbcache` supposed to be committed? If not, we should make sure that things are set up so that they cannot be committed (starting from `.gitignore` and possibly adding CI checks to verify).

File: idaes_examples/notebooks/docs/surrogates/alamo_run.alm
Comment (andrewlee94) [reply idaes_examples/notebooks/docs/surrogates/alamo_run.alm:23]: You are going about it the wrong way - you should just revert the changes you made through git. @lbianchi-lbl is probably best to help you through that (he is better at git than I am).

Comment (luohezhiming) [reply idaes_examples/notebooks/docs/surrogates/alamo_run.alm]: I find the way to revert this change, it just happen when I run `idaesx build`.

Comment (andrewlee94) [reply idaes_examples/notebooks/docs/surrogates/alamo_run.alm]: I think you also need to remove all the things in the nbcache as well - they should not be part of this PR (as @lbianchi-lbl noted we should have these listed in the `.gitignore` so that they don;t get accidentally included).

Also, you might need to add the new notebooks to an indexing page - @dangunter is the one to talk to about what has to be done there (I could not find any documentation of that).

Comment (luohezhiming) [reply idaes_examples/notebooks/docs/surrogates/alamo_run.alm]: I followed the README-developer.md, and the nbcahe files are created by running `idaesx build`. In the instruction, it is said that "Note that the cache for the documentation will be updated to contain the new output cells, which will modify files in idaes_examples/notebooks/nbcache; these files should also be committed and pushed."

**Comment by lbianchi-lbl** [comment]
This is waiting on a reply from us (i.e. @dangunter) on whether the nbcache files should be committed. Also, idoes @andrewlee94 have any insight on whether this should be in the May release?

**Comment by dangunter** [comment]
We can discuss, but I was thinking we do commit the cache files to speed up builds.

**Comment by andrewlee94** [comment]
@luohezhiming There still appear to be some files in this PR related to surrogates examples that should not be getting changed here.

**Comment by andrewlee94** [comment]
@luohezhiming I have also been talking to @dangunter about the cache files as they are not really something that can be reviewed. His solution (for now at least) is to remove the cached files from the PR:

`git rm -r idaes_examples/notebooks/nb_cache`

**Comment by luohezhiming** [comment]
> @luohezhiming I have also been talking to @dangunter about the cache files as they are not really something that can be reviewed. His solution (for now at least) is to remove the cached files from the PR:
> 
> `git rm -r idaes_examples/notebooks/nb_cache`

@andrewlee94 I've remove those nbcache files. I wondered next time do I still need to run `idaesx build` to update the cache?

**Comment by andrewlee94** [comment]
@luohezhiming That is a question for @dangunter

**Comment by dangunter** [comment]
In PR #44 we have decided to drop the cache files. But this will affect this PR. I can help with changes once #44 is merged.

File: idaes_examples/notebooks/docs/surrogates/alamo_run.alm
Comment (lbianchi-lbl) [reply idaes_examples/notebooks/docs/surrogates/alamo_run.alm]: In addition to the changes in how caching works from #44, I've updated `.gitignore` so that this ALAMO file is also not tracked by Git.

**Comment by lbianchi-lbl** [comment]
I don't know/remember if this notebook is supposed to be included in the docs (i.e. moved to the `docs` directory and added to `_toc.yml`), or not (i.e. moved to `active`?). @luohezhiming @andrewlee94 @MAZamarripa @dangunter thoughts?

**Comment by andrewlee94** [comment]
@lbianchi-lbl I am not quite sure what it means for something to be moved to `active` but not to be in the docs. I would have thought we wanted all examples to be listed in the docs (we are moving these things so that they are easy to find); for things like this my thought is that we want an "examples" section in the docs for holding non-tutorial materials that are provided for users to look at.

**Review by bpaul4 — APPROVED** [review]
The notebooks look good, although the supercritical steam example contains a couple long lists of `WARNING: model contains export suffix` output lines that would be good to clean up if possible.

**Comment by dangunter** [comment]
@andrewlee94 the "active" directory was intended for some of the examples John Eslick said didn't need to be in the docs. The term "active" is meant to indicate that they are still included in the tests.

**Pull request merged**
